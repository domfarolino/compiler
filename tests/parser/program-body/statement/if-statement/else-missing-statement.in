program dominic is

  integer dominic;

begin
  dominic := 10;

  if (dominic == 10) then
    dominic := 11;
  else
  end if;

end program.
