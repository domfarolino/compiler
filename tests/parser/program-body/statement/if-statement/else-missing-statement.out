[1;31mLine 11: Expected at least one statement after 'else' in if statement, but got: 'end'
Line 11: Error parsing program body
[0m